test/unit/test_paired_loader.py

```diff
@@ -0,0 +1,136 @@
+"""
+Tests functionality of the PairedDataLoader
+"""
+import pytest
+
+from deepreg.dataset.loader.h5_loader import H5FileLoader
+from deepreg.dataset.loader.paired_loader import PairedDataLoader
+
+# assign values to input vars
+data_dir_path = "./data/test/h5/paired/test"
+sample_label = "sample"
+seed = (8, 8, 8)
```

Comment:
the seed should be int lol ? like 0 ?
and the shapes in the following should be (8, 8, 8)